utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -394,20 +577,105 @@ inline void convert_pkd3_to_pln3(Rpp8u *input, RpptDescPtr descPtr)
     free(inputCopy);
 }
 
-inline void read_image_batch_opencv(Rpp8u *input, RpptDescPtr descPtr, string imageNames[])
+// Opens a folder and recursively search for .jpg files
+void open_folder(const string& folderPath, vector<string>& imageNames, vector<string>& imageNamesPath)
+{
+    auto src_dir = opendir (folderPath.c_str());
+    struct dirent* entity;
+    std::string fileName = " ";
+
+    if (src_dir == nullptr)
+        std::cerr << "\n ERROR: Failed opening the directory at " <<folderPath;
+
+    while((entity = readdir (src_dir)) != nullptr)
+    {
+        string entry_name(entity->d_name);
+        if (entry_name == "." || entry_name == "..")
+            continue;
+        fileName = entity->d_name;
+        std::string filePath = folderPath;
+        filePath.append("/");
+        filePath.append(entity->d_name);
+        fs::path pathObj(filePath);
+        if(fs::exists(pathObj) && fs::is_directory(pathObj))
+        {
+            open_folder(filePath, imageNames, imageNamesPath);
```

Comment:
Remove braces if there is only statement under if conditions

Reply:
Done